Core/Resources/ErrorMessages.resx.bak

```diff
@@ -0,0 +1,222 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do we need this file? It is created after a change in the text editor, but it is not needed in the project